drivers/wifi/nrf700x/src/fmac_main.c

```diff
@@ -281,9 +281,21 @@ void nrf_wifi_event_get_reg_zep(void *vif_ctx,
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
 	fmac_dev_ctx = rpu_ctx_zep->rpu_ctx;
 
+	if (fmac_dev_ctx->alpha2_valid) {
```

Comment:
This isn't thread-safe, e.g., calling GET_REG from different threads. The original code for alpha2 itself isn't, so, need to fix that and then the same mechanism can be re-used.

Reply:
Added locking meachanism to sync cmd/event.